tests/faker.py

```diff
@@ -0,0 +1,23 @@
+def fake_creds():
+    return {
+        'FAKE_CHANNEL': 'C12942JF92'
+    }
+
+
+class FakeClient:
+    def __init__(self):
+        self.rtm_messages = []
+
+    def rtm_send_message(self, channel, message, attachments=None):
```

Comment:
Got it. But don't forget that this is a mock and the library has no idea of what goes on here. The library uses it even if it default to None, but our mock doesn't use it at all.